frontend/src/components/pages/topics/CreateTopicModal/CreateTopicModal.tsx

```diff
@@ -590,15 +600,15 @@ export function RatioInput(p: {
             onChange={x => p.onChange(x / 100)}
             tooltip={{ formatter: null }}
         />
-        <InputNumber
+        <NumberInput
             min={0} max={100}
-            value={Math.round(p.value * 100)}
+            value={Math.round(p.value * 100) + '%'}
```

Comment:
This one is a bit hard to test as I'm not getting `RATIO` config entry type cc @weeco . Is it still used?

I've tried to test by emulating this for other values and it seems to work.

Reply:
So far, I have not seen a ratio type entry yet. So it should be ok for now; if there is an issue, we will find it later when there is an actually reproducible situation